docs/user-journeys/users/cluster-operator/intermediate.md

```diff
@@ -18,7 +18,7 @@ If you are a cluster operator looking to expand your grasp of Kubernetes, this p
 
 ## Work with Ingress, Networking, Storage, Workloads
 
```

Comment:
All the other heading use sentence case, not title case. So I think this heading should be:

Work with ingress, networking, storage, workloads